### ISSUE TITLE
Тени в индикаторе Total Delta/

### ISSUE THREAD

**Issue by AlexandrBik** [SBProX/SBProX-Dev#1680]
Добрый день! Очень важная деталь, которая катастрофически необходима, для тех кто торгует объемный анализ и использует индикатор Total Delta горизонтальный. Это Тени в самом индикаторе, а не только тело столбца объема. Именно тени дают больше информации, для определения лимитных ордеров и прочее.
 Большая просьба, сделайте возможность отображения тени. 
Файл прилагаю, взято от
https://k-pavel.ru/indikator-delta-5-0-quik-pit-vol

![2020-11-16_12-34-02](https://user-images.githubusercontent.com/52658923/99236975-eadaf680-2808-11eb-939e-a8ecbfbafe80.png)
ume-divergence/

**Comment by weeclemans** [comment]
без данных с Level2 такое сделать будет сложновато

**Comment by Jewel177** [comment]
Давно хотела написать про эту фишку. Уважаемые разработчики, добавьте пожалуйста тени для дельты. Это очень информативный показатель, прикладываю торговую ситуацию..для наглядности.
![дельта](https://user-images.githubusercontent.com/56072909/99499968-babb6100-298a-11eb-9436-a23aa1ce86f8.png)

**Comment by AlexandrBik** [comment]
> Давно хотела написать про эту фишку. Уважаемые разработчики, добавьте пожалуйста тени для дельты. Это очень информативный показатель, прикладываю торговую ситуацию..для наглядности.

Да! Это сделает индикатор полезнее. Такого нет в большинства платформ.

**Comment by AndrewAK86** [comment]
Добавьте пожалуйста тени к индикатору дельта, очень их не хватает для полноценного анализа. Спасибо большое за понимание!

**Comment by AndrewAK86** [comment]
Прошу добавить дельту в виде свечей.
